app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -303,10 +314,18 @@ object TelemetryWrapper {
 
     @JvmStatic
     fun stopMainActivity() {
-        TelemetryHolder.get()
-                .queuePing(TelemetryCorePingBuilder.TYPE)
-                .queuePing(TelemetryEventPingBuilder.TYPE)
-                .scheduleUpload()
+        if (shouldSendMobileMetrics) {
+            TelemetryHolder.get()
+                    .queuePing(TelemetryCorePingBuilder.TYPE)
+                    .queuePing(TelemetryEventPingBuilder.TYPE)
+                    .queuePing(TelemetryMobileMetricsPingBuilder.TYPE)
+                    .scheduleUpload()
```

Comment:
This might be able to be simplified with `.also`. Let me know what you think!

```
 TelemetryHolder.get()
                    .queuePing(TelemetryCorePingBuilder.TYPE)
                    .queuePing(TelemetryEventPingBuilder.TYPE)
                    .also { if (shouldSendMobileMetrics) queuePing(TelemetryMobileMetricsPingBuilder.TYPE) }
                    .scheduleUpload()
```